docs/news/index.qmd

```diff
@@ -2,6 +2,163 @@
 
 This page provides the release notes associated with each release of RStudio and Posit Workbench. Please contact customer support (<a href="mailto:support@posit.co">support@posit.co</a>) for questions about the described changes.
 
+## RStudio 2024.04.0
+
+**"Chocolate Cosmos"**
+
+>Date: 2024-04-29
+
+### New
+
+#### RStudio
+
+- RStudio now requires R 3.6.0 or newer. (#14210)
```

Comment:
super nit: empty line between the RStudio subheading and list items here and in the Fixed section below

Reply:
Since this is a rendered quarto document, the newline differences are rendered out, so I'm just going to leave them since they're copied from the news file